docs/sources/developers/http_api/auth.md

```diff
@@ -29,6 +29,8 @@ If you use Grafana v9.1 or newer, use service accounts instead of API keys. For
 
 ## List API keys
 
+### DEPRECATED
```

Comment:
nit: for this and the following doc changes - I think using something like `{{% admonition type="warning" %}}` would be more appropriate than adding a new header.